src/kfs_lib/kfs_init.c

```diff
@@ -7,6 +7,7 @@ void *kfs_init(struct fuse_conn_info *connection) {
     if (XGLFS_STATE->gluster_api) {
         log_info("   gluster sync enabled");
         res = xglfs_init(connection);
+        read_cluster();
```

Comment:
if we are not able to donwload content we should exit

Reply:
fixed 9566f3f